<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playfair Encryption</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="gbCurrentSettings">
      <property name="title">
       <string>Current matrix</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lblCurrentMatrix">
           <property name="text">
            <string>Matrix:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtCurrentMatrix">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblMatrixSize">
           <property name="text">
            <string>Matrix size:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="txtMatrixSize">
           <property name="text">
            <string>6x6</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnChangeMatrix">
           <property name="text">
            <string>Change matrix</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="TabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="TabEncode">
       <attribute name="title">
        <string>Encode</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="TabEncode_lblInputText">
          <property name="text">
           <string>Input text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="TabEncode_txtInputText"/>
        </item>
        <item>
         <widget class="QLabel" name="TabEncode_lblOutputText">
          <property name="text">
           <string>Output text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="TabEncode_txtOutputText"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnEncode">
            <property name="text">
             <string>Encode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabDecode">
       <attribute name="title">
        <string>Decode</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="TabDecode_lblInputText">
          <property name="text">
           <string>Input text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="TabDecode_txtInputText"/>
        </item>
        <item>
         <widget class="QLabel" name="TabDecode_lblOutputText">
          <property name="text">
           <string>Output text:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="TabDecode_txtOutputText"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnDecode">
            <property name="text">
             <string>Decode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <action name="actionMatriz">
   <property name="text">
    <string>Set key matrix</string>
   </property>
  </action>
  <action name="MenuOptions_ChiperSettings">
   <property name="text">
    <string>Chiper settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
